docs/source/getting-started/content/proc_starting-monitoring-alerting-telemetry.adoc

```diff
@@ -0,0 +1,16 @@
+[id="starting-monitoring-alerting-telemetry_{context}"]
+= Starting Monitoring, Alerting, and Telemetry
+
+In order to make sure {prod} can run on a typical laptop, some resource–heavy services get disabled by default.
+One of these is Prometheus and all the related monitoring, alerting, and telemetry functionality, with the latter being
```

Comment:
Apologies if this was unclear, but we really do prefer one sentence per line -- this means a complete sentence on each line.

This practice makes diffs much cleaner, which is why we try to do this as much as possible in documentation.